src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts

```diff
@@ -224,32 +231,34 @@ describe('HmctsGlobalHeaderComponent', () => {
       })
     ).subscribe(items => {
       expect(items).toEqual([component.items[0]]);
-      done();
     });
   });
 
-  it('filters out menu items for which not all features are enabled', (done) => {
+  it('filters out menu items for which not all features are enabled', async () => {
```

Comment:
@johnbenjamin-hmcts - see previous comment on `(done)`.

Reply:
Thank you @Daniel-Lam - I have now reverted the changes.